wire-ios-sync-engine/Source/UserSession/ZMUserSession/ZMUserSession.swift

```diff
@@ -384,6 +387,54 @@ public final class ZMUserSession: NSObject {
         self.contextStorage = contextStorage
         self.recurringActionService = recurringActionService
         self.dependencies = dependencies
+
+        super.init()
+
+        setupAnalyticsSession()
+
+        NotificationCenter.default.addObserver(
+            self,
+            selector: #selector(handleAppDidBecomeActive),
+            name: UIApplication.didBecomeActiveNotification,
+            object: nil
+        )
+
+    }
+
+    private func setupAnalyticsSession() {
+        guard let config = dependencies.analyticsSessionConfiguration,
+              let countlyKey = config.countlyKey,
+              let host = config.host else {
+            return
+        }
+
+        let teamInfo: TeamInfo? = {
+            guard let team = selfUser.membership?.team,
+                  let teamID = team.remoteIdentifier?.uuidString else {
+                return nil
+            }
+
+            let teamRole = selfUser.teamRole.analyticsValue
+            let teamSize = team.members.count
+
+            return TeamInfo(id: teamID, role: teamRole, size: teamSize)
+        }()
+
+        let analyticsUserProfile = AnalyticsUserProfile(
+            analyticsIdentifier: selfUser.remoteIdentifier.uuidString,
+            teamInfo: teamInfo,
+            contactCount: selfUser.membership?.team?.members.count
```

Comment:
What is the point of contact count if it's the same as the team size in the team info?

Reply:
I don't think we need contact count tbh. I got rid of it here: https://github.com/wireapp/wire-ios/commit/86b4f83e94d783ece3d51466a4d8b17ddc50c0ef